src/overlays/actors/ovl_En_Kaizoku/z_en_kaizoku.c

```diff
@@ -65,118 +136,1921 @@ const ActorInit En_Kaizoku_InitVars = {
+    {
+        COLTYPE_NONE,
+        AT_NONE,
+        AC_ON | AC_TYPE_PLAYER,
+        OC1_ON | OC1_TYPE_ALL,
+        OC2_TYPE_1,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK4,
+        { 0x00000000, 0x00, 0x00 },
+        { 0xF7CFFFFF, 0x00, 0x00 },
+        TOUCH_NONE | TOUCH_SFX_NORMAL,
+        BUMP_ON | BUMP_HOOKABLE,
+        OCELEM_ON,
+    },
     { 20, 50, 0, { 0, 0, 0 } },
 };
 
 // static ColliderQuadInit sQuadInit = {
 static ColliderQuadInit D_80B8ABA8 = {
-    { COLTYPE_NONE, AT_ON | AT_TYPE_PLAYER | AT_TYPE_ENEMY, AC_NONE, OC1_NONE, OC2_NONE, COLSHAPE_QUAD, },
-    { ELEMTYPE_UNK4, { 0xF7CFFFFF, 0x00, 0x00 }, { 0x00000000, 0x00, 0x00 }, TOUCH_ON | TOUCH_SFX_NORMAL | TOUCH_UNK7, BUMP_NONE, OCELEM_NONE, },
+    {
+        COLTYPE_NONE,
+        AT_ON | AT_TYPE_PLAYER | AT_TYPE_ENEMY,
+        AC_NONE,
+        OC1_NONE,
+        OC2_NONE,
+        COLSHAPE_QUAD,
+    },
+    {
+        ELEMTYPE_UNK4,
+        { 0xF7CFFFFF, 0x00, 0x00 },
+        { 0x00000000, 0x00, 0x00 },
+        TOUCH_ON | TOUCH_SFX_NORMAL | TOUCH_UNK7,
+        BUMP_NONE,
+        OCELEM_NONE,
+    },
     { { { 0.0f, 0.0f, 0.0f }, { 0.0f, 0.0f, 0.0f }, { 0.0f, 0.0f, 0.0f }, { 0.0f, 0.0f, 0.0f } } },
 };
 
-#endif
+u32 D_80B8ABF8[] = {
+    0x0600F8E4, 0x0600EF9C, 0x0600E1C8, 0x0600DBE4, 0x060058B8, 0x06004860, 0x06002730,
+    0x06001578, 0x06001390, 0x060003CC, 0x06000F5C, 0x0600E8BC, 0x0600ED1C, 0x06005644,
+    0x0600F288, 0x060043E4, 0x06003A3C, 0x06002BA0, 0x06001E9C,
+};
+
+u8 D_80B8AC44[] = {
+    0x00, 0x00, 0x02, 0x00, 0x00, 0x02, 0x02, 0x02, 0x02, 0x02,
+    0x02, 0x02, 0x00, 0x02, 0x02, 0x02, 0x02, 0x00, 0x02, 0x00,
+};
+
+Color_RGBA8 D_80B8AC58 = { 0xC8, 0xA0, 0x78, 0xFF };
+Color_RGBA8 D_80B8AC5C = { 0x82, 0x5A, 0x32, 0xFF };
+
+Vec3f D_80B8AC60 = { 0.0f, -1.5f, 0.0f };
+Vec3f D_80B8AC6C = { 0.0f, -0.2f, 0.0f };
+Vec3f D_80B8AC78 = { 300.0f, 0.0f, 0.0f };
+Vec3f D_80B8AC84 = { 0.0f, -3000.0f, 0.0f };
+Vec3f D_80B8AC90 = { 400.0f, 0.0f, 0.0f };
+Vec3f D_80B8AC9C = { 1600.0f, -4000.0f, 0.0f };
+Vec3f D_80B8ACA8 = { -3000.0f, -2000.0f, 1300.0f };
+Vec3f D_80B8ACB4 = { -3000.0f, -2000.0f, -1300.0f };
+Vec3f D_80B8ACC0 = { 1000.0f, 1000.0f, 0.0f };
+
+u32 D_80B8ACCC[] = {
+    0x060093B8, 0x06009978, 0x0600A0F8, 0x06009978, 0x00000000,
+};
+
+extern AnimationHeader D_060058B8;
+
+extern FlexSkeletonHeader D_0600D828;
+
+void EnKaizoku_Init(Actor* thisx, PlayState* play) {
+    EnKaizoku* this = THIS;
+    s32 pad;
+    Player* player = GET_PLAYER(play);
+    EffectBlureInit1 blureInit;
+
+    this->actor.hintId = 0x54;
+    this->actor.targetMode = 3;
+    this->actor.colChkInfo.mass = 80;
+    ActorShape_Init(&this->actor.shape, 0.0f, ActorShadow_DrawFeet, 0.0f);
+    this->unk_2CA = this->actor.world.rot.z;
+    this->actor.colChkInfo.health = 8;
+    this->unk_2D4 = this->actor.params & 0x3F;
+    this->unk_2BC = (this->actor.params >> 6) & 0x7F;
```

Comment:
Kenix is gonna ask you got make macros from these, where 2BC is switch flags because it gets passed to Flags_GetSwitch later down.